### DISCUSSION TITLE
Support of panoptic segmentation

### DISCUSSION THREAD

**Discussion by zheng034** [deepchecks/deepchecks#1944] [Ideas]
Hi do you have any plan to support panoptic segmentation for data integrity and train test validation? Thank yoU!

**Comment by noamzbr** [comment]
Hi @zheng034! We are now working on adding semantic segmentation to deepchecks.vision (You can follow https://github.com/deepchecks/deepchecks/issues/1939 to know when we're done adding this new task type). 
We haven't planned to add other types of segmentation ourselves yet, but we'd love to hear what special checks you'd like to see. You can open an issue with feature suggestions. 
You are also welcome to add it yourself! You can read our guide on adding new custom tasks [here](https://docs.deepchecks.com/stable/user-guide/vision/auto_tutorials/plot_custom_task_tutorial.html).
